### PR TITLE
feat/auto-release

### PR DESCRIPTION
**Pull request by brittonhayes**
## What does this do?

Adds automated semantic release github action.

### CODE REVIEW

File: .github/workflows/github-release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        - name: Setup Node.js
+          uses: actions/setup-node@v2
+          with:
+            node-version: 'lts/*'
+        - name: Install dependencies
+          run: npm install
+        - name: Release
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          run: npx semantic-release
```
Comment (bagelbits) [inline .github/workflows/github-release.yml]:
```suggestion
          run: npx semantic-release

```

File: .github/workflows/container-release.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: container-release
```
Comment (bagelbits) [inline .github/workflows/container-release.yml]: In one of these release scripts, should we also run the release script that populates the database?

Also should this replace the other release flow?

**Review by bagelbits — APPROVED** [review]
A couple of questions but otherwise looks solid.

File: .github/workflows/container-release.yml (hunk 2 above)
Comment (brittonhayes) [reply .github/workflows/container-release.yml]: Yeah I think that's a good idea. Just get it all goin in one place.

**Pull request merged**